<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PRATIK" failures="0" tests="1" name="seleniumDay23_Assertion.SoftAssertDemo" time="18.525" errors="1" timestamp="5 Apr 2022 15:25:45 GMT">
  <testcase classname="seleniumDay23_Assertion.SoftAssertDemo" name="userAddFlow" time="18.525">
    <error message="The following asserts failed:
	expected [file:///D:/Study/1.SDET/JAVA/OfflineWebsite/pages/examples/dashboardDEmo.html] but found [file:///D:/Study/1.SDET/JAVA/OfflineWebsite/pages/examples/dashboard.html],
	expected [true] but found [[[ChromeDriver: chrome on WINDOWS (47484eda1468a539221b4f0dc27687db)] -&gt; xpath: //input[@id=&#039;Male&#039;]]],
	Wrong Alert Message on alert box expected [User Added Successfully. You can see added user.] but found [User Added Successfully. You can not see added user.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [file:///D:/Study/1.SDET/JAVA/OfflineWebsite/pages/examples/dashboardDEmo.html] but found [file:///D:/Study/1.SDET/JAVA/OfflineWebsite/pages/examples/dashboard.html],
	expected [true] but found [[[ChromeDriver: chrome on WINDOWS (47484eda1468a539221b4f0dc27687db)] -> xpath: //input[@id='Male']]],
	Wrong Alert Message on alert box expected [User Added Successfully. You can see added user.] but found [User Added Successfully. You can not see added user.]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
	at seleniumDay23_Assertion.SoftAssertDemo.userAddFlow(SoftAssertDemo.java:101)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- userAddFlow -->
</testsuite> <!-- seleniumDay23_Assertion.SoftAssertDemo -->
